vars/kibanaPipeline.groovy

```diff
@@ -97,7 +97,7 @@ def uploadCoverageArtifacts(prefix, pattern) {
 def withGcsArtifactUpload(workerName, closure) {
   def uploadPrefix = "kibana-ci-artifacts/jobs/${env.JOB_NAME}/${BUILD_NUMBER}/${workerName}"
   def ARTIFACT_PATTERNS = [
-    'target/kibana-*',
+    'target/kibana-*/**/*',
```

Comment:
@MadameSheema can't we be more specific about what are the folders/types of files we need to add for the Cypress screenshots?  Or do we really need `kibana-*/**/*`?

Reply:
I can be more specific, @patrykkopycinski also told me the same :)
As that lines of code were not added by us (SIEM team) I didn't want to change it to much originally.